php-nginx/usr/local/sbin/postfix.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# call "postfix stop" when exiting
+trap "{ echo Stopping Postfix; /usr/sbin/postfix stop; exit 0; }" EXIT
+
+source /etc/sysconfig/network
+
+/usr/libexec/postfix/aliasesdb
+/usr/libexec/postfix/chroot-update
+
+if [ -n "${POSTFIX_RELAY_USER}" ]; then
```

Comment:
Fixes incoming from @fratcliffe but this would be `SENDMAIL_RELAY_USER`, we think.